core/src/main/java/com/linecorp/armeria/internal/common/util/StringUtil.java

```diff
@@ -56,7 +57,7 @@ public static String toString(long num) {
     }
 
     public static Boolean toBoolean(String s, boolean errorOnFailure) {
-        final Boolean result = stringToBoolean.get(Ascii.toLowerCase(s));
+        final Boolean result = stringToBoolean.get(s);
```

Comment:
> (https://github.com/line/armeria/pull/5319#discussion_r1457089903) I'm unsure if `tRUE` or `TrUe` can be treated as a boolean value.
> How about only supporting the values defined in stringToBoolean?

FYI to maintainers) based on above comment, we decide to support only `true, TRUE, 1` on `getBoolean, containsBoolean` so I think it's good to share this change on release note or somewhere?

Cause it maybe `breaking change` on some user that using value like `tRuE` on `getBoolean, containsBoolean` 😅

Reply:
It may be not a breaking change but a bug. That said, it would be worth adding an example code for the behavior of `getBoolean`, `containsBoolean`.

Reply:
> example code for the behavior of getBoolean, containsBoolean.

aha~ I agree with this. then should we add example code on `HttpHeaderGetters, QueryParamGetters # getBoolean, containsBoolean`?

I'll add comment!